spec/abstractops.html

```diff
@@ -1492,7 +1492,7 @@ <h1>ParseTemporalTimeZoneString ( _isoString_ )</h1>
         1. Set _seconds_ to ! ToIntegerOrInfinity(_seconds_).
         1. If _fraction_ is not *undefined*, then
           1. Set _fraction_ to the string-concatenation of the previous value of _fraction_ and the string *"000000000"*.
-          1. Let _nanoseconds_ be the String value equal to the substring of _fraction_ from 1 to 10.
+          1. Let _nanoseconds_ be the String value equal to the substring of _fraction_ consisting of the code units with indices 0 (inclusive) through 9 (exclusive).
```

Comment:
Wording here is taken from `ParseTimeZoneOffsetString`:

https://github.com/tc39/proposal-temporal/blob/2a81fbca981eb684f1dc10ec479b5a8e872d8c37/spec/timezone.html#L516

Reply:
As per the conversation in https://github.com/tc39/proposal-temporal/pull/1830#discussion_r710402655, we determined that the wordier variant was unnecessary, so this can just be "the substring of _fraction_ from 0 to 9".

Reply:
Got it, updated :+1: